### DISCUSSION TITLE
How to change the date format displayed?

### DISCUSSION THREAD

**Discussion by accnetdev** [rejetto/hfs#394] [Q&A]
Hi, I want to get the date format to 24h and DD/MM/YY how to do it please?

**Comment by rejetto** [comment]
i'm sorry, i forgot about this message.
do you still need this?

**Reply by PH5HP** [reply]
Hi,

For example we use this format in the Netherlands, but I am sure that there are more countries who uses this Date-format...

**Reply by rejetto** [reply]
just for information, i'm leaving the formatting of date and time to the browser, and it's up to how you configured your browser. By default your browser will refer to how you configured your operating system.

**Reply by PH5HP** [reply]
Hi,

Seems for me the right decision.
